package.json

```diff
@@ -25,5 +25,6 @@
   "homepage": "https://github.com/emn178/hi-base32",
   "bugs": {
     "url": "https://github.com/emn178/hi-base32/issues"
-  }
+  },
+  "typings": "./hi-base32.d.ts"
```

Comment:
Rename file to `index.d.ts`, TypeScript looks up this path in default. 
And look like document also uses `types` in default.